src/components/10-atoms/progress-bar/pw.ui.test.js

```diff
@@ -0,0 +1,55 @@
+const host = process.env.TEST_HOST_STORYBOOK_URL;
+const tag = 'axa-progress-bar';
+
+describe('Progress Bar', () => {
+  it('should render small', async () => {
+    await openProgressBar();
+
+    const fontSize = await page.$eval(
+      'p',
+      el => window.getComputedStyle(el).fontSize
+    );
+
+    expect(fontSize).toBe('16px');
+  });
+
+  it('should write text correctly', async () => {
+    await openProgressBar();
+
+    expect(
+      await page.textContent(
+        'article > div.a-progress-bar--label > axa-text > p'
```

Comment:
Better use classes only, as far as possible. This test will otherwise break all the time when something internal gets changed.